src/app/models/user.server.model.ts

```diff
@@ -75,6 +75,7 @@ const compileUserModel = (db: Mongoose) => {
         payment: Boolean,
         children: Boolean,
         postmanSms: Boolean,
+        fasterDownloads: Boolean,
```

Comment:
Shouldn't need this change since we're using growthbook (FE flag)

Reply:
Fixed!